config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=21.7
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=21.7.0.0
+VERSION_LONG=21.7.0.1
```

Comment:
Note: The title of the PR was misleading as it had a typo suggesting this was `27.1.0.1` (instead of `21.7.0.1` like it is here in reality). Took the liberty to update the PR title accordingly.

Reply:
Shoot, didn't realize that auto-merge was on… which meant that the PR got auto-merged before I had time to change the title, which means the merge commit still contains a commit message mentioning `27.1.0.1` instead of `21.7.0.1`. Oh well.